src/memory_pool.c

```diff
@@ -99,6 +145,16 @@ static umf_result_t umfPoolCreateInternal(const umf_memory_pool_ops_t *ops,
         goto err_pool_init;
     }
 
+    // Set default property "name" to pool if exists
+    for (int i = 0; i < UMF_DEFAULT_SIZE; i++) {
+        if (CTL_DEFAULT_ENTRIES[i][0] != '\0' &&
+            strstr(CTL_DEFAULT_ENTRIES[i], umfPoolGetName(pool))) {
```

Comment:
Please cache umfPoolName, as you can change it latter on with ctl call, and all following default entries will not work

Reply:
Now should work as designed, I've added test for that.